assets/scss/modules/_tables.scss

```diff
@@ -1 +1,117 @@
+////
+///	Tables
+/// @group Tables
+/// @author gbaamonde@makingsense.com
+////
+/// Table Component -
+/// This is a simple easy-to-use table component.
+/// @example markup
+///   <table class="dp-c-table">
+///     <caption>asdasdasd</caption>
+///     <thead>
+///       <tr>
+///         <th>
+///           Column 1
+///         </th>
+///         <th>
+///           Column 2
+///         </th>
+///       </tr>
+///     </thead>
+///     <tfoot>
+///       <tr>
+///         <td colspan="2">
+///           Single column for <tfoot>
+///         </td>
+///       </tr>
+///     </tfoot>
+///     <tbody>
+///       <tr>
+///         <td>
+///           Content 1
+///         </td>
+///         <td>
+///           Content 2
+///         </td>
+///       </tr>
+///     </tbody>
+///   </table>
 
+@mixin dp-c-table {
+
+  .dp-c-table {
+    width: 100%;
+    font-size: $dp-sizing--lvl2;
+    border-collapse: collapse;
+    border-spacing: $dp-spaces--lv0;
+    background: $dp-color-white;
+
+    th {
+      background: $dp-color-white;
+      border-top: 1px solid $dp-color-silver;
+      border-bottom: 1px dashed $dp-color-lightgrey;
+      font-size: $dp-sizing--lvl1;
+      font-weight: normal;
+      color: $dp-color-grey;
+      height: 40px;
+      line-height: 18px;
```

Comment:
There are no constants for these values?

Reply:
the line-height is used according to the size of the typography, we do not have all the line-height defined

Reply:
In that case, it should not be a kind of calculus? 

I mean, suppose that you change the value of _$dp-sizing--lvl1_, for example from _15px_ to _60px_, in that case, a _line-height_ of _18px_ will be wrong.

I am missing something?